app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.kt

```diff
@@ -905,4 +792,70 @@ class TeiDashboardMobileActivity :
             return intent
         }
     }
+
+    private fun setupMoreMenu() {
```

Comment:
I think the nomenclature of this method could be clearer, what do you think of renaming it to something like setUpMoreOptionsMenu?